proposal-v1.md

```diff
@@ -0,0 +1,221 @@
+- @jpsantanna, crypto enthusiast since 2017 and follows Decred progress
+  since 2018. Has been managing and creating content for Decred Brazil Instagram
+  and Twitter accounts since November 2021.
+- @edsondecre has worked on several events for Decred, such as Campus Party,
+  Bitconf, and other meetups, with a high-quality community management
+  experience. Edson was able to include Decred in several exchanges like:
+  Bitcambio (https://www.bitcambio.com.br/) and ComprarBitcoin
+  (https://comprarbitcoin.com.br/).
+- @eric_vieira, moderator of DECRED Brasil (Facebook) since February 6, 2017,
+  has been actively explaining technical parts of the project on Facebook
+  Group and also organized a speech about Decred project in Centro Universitário
+  Estácio Juiz de Fora.
+
+The Decred Brazil team will be composed by 3 levels of contributors, with
+@victorguedes as the supervisor. The roles are detailed below:
+
+#### Level 1
+
+Rate: $10/hr
+
+The Level 1 is the first stage of the Decred Brazil contribution journey. People
+who proved to be able to actively contribute over the Decred community, and take
+some entry-level responsibilities, which are described below. A Level 1
+contributor will have to be supervised by a higher-level contributor, in order
+to ensure the correct information, design and quality is published.
+
+##### Expected Roles
+
+- Content production for small assets like images, gifs, post templates,
+  banners, short videos (Instagram reels and stories, TikTok videos, Youtube
+  shorts);
+- Text creation: Twitter threads, posts description, small content translation;
+- Support at events, helping to organize, engage and publicize the event;
+- Collect community feedbacks;
+- Administrate social media pages;
+
+#### Level 2
+
+Rate: $20/hr
+
+The Level 2 represents a more experienced stage, where contributors are ready to
+produce content on an autonomous way, i.e. no supervision required. A Level 2
+contributor have proved commitment, with a noticeable presence on both
+international and BR community, producing relevant contents, attending relevant
+events and meetings.
+
+##### Expected Roles
+
+- Level 1 expected roles;
+- Noticeable presence on events, coordinating at least one aspect (speech,
+  interaction dynamics...);
+- Complex content production like long videos, podcasts, interviews, articles,
+  tutorials;
+- Support and help the community to handle some possible issues, as well as
+  report the feedbacks using the correct mean of communication with Decred devs
+  (github issues and discussions, appropriate matrix channel...);
+- Review and evaluate social media metrics, analyse the performance of our
+  marketing campaigns, and provide a report document periodically.
+- Supervise Level 1 contributors
+
+#### Level 3
+
+Rate: $30/hr
+
+The last and top level of Decred Brazil contribution journey. A contributor from
+this level is experienced enough to coordinate the team along with other Level 3
+contributors.
```

Comment:
Level 3 must be an influent character on the crypto scenario, experienced with marketing campaigns and metrics....